meta-balena-common/recipes-core/initrdscripts/files/resindataexpander

```diff
@@ -9,7 +10,12 @@ get_part_table_type() {
     parted -s "$1" print | grep "Partition Table" | tr -d " " |cut -d ":" -f2
 }
 
+get_device_sector_count() {
+    parted -m /dev/nvme0n1 unit s print | grep /dev/ | cut -d ":" -f2 | tr -d "s"
```

Comment:
Probably you didn't want to hardcode nvme0n1 here?

Reply:
Probably not :grimacing: